jasmine/src/bowling.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
+
+var BowlingScorecard = function() {
+  this.knockedDownPins = 0;
+  this.frame = []
+  this.total = []
+  this.frameCount = 0
+};
+
+BowlingScorecard.prototype.getCurrentRollScore = function() {
+  return this.knockedDownPins;
+};
+
+BowlingScorecard.prototype.rollScore = function(knockedDownPins) {
```

Comment:
Well done on making your functions short and following the single responsibility principle.